src/Serilog.Sinks.OpenTelemetry/Helpers/OpenTelemetryEnvironment.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Serilog.Sinks.OpenTelemetry;
+
+namespace Serilog.Helpers;
+
+internal static class OpenTelemetryEnvironment
+{
+    public static void Configure(BatchedOpenTelemetrySinkOptions options)
+    {
+        options.Endpoint = Endpoint;
+        options.Headers = Headers;
+        options.ResourceAttributes = ResourceAttributes;
```

Comment:
When `OTEL_EXPORTER_OTLP_ENDPOINT` is provided, and the protocol is determined to be http/protobuf, if the value doesn't end with `/v1/logs` it will be necessary to tack this on. (In a near-future version of the sink this will be fixed, but some more pieces need to come together around it.)